webpack.config.js

```diff
@@ -55,11 +55,16 @@ module.exports = {
           //https://glenmaddern.com/articles/css-modules
           //https://github.com/css-modules/css-modules
             {
-                test: [/\.css$/, /\.scss$/, /\.sass$/],
-                loader: ExtractTextPlugin.extract({
-                    fallback: 'style-loader',
-                    use: 'style-loader!css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!sass-loader!postcss-loader'
-                })
+                test: /\.css$/,
+                loader: 'style-loader'
+            }, 
+            {
+                test: /\.css$/,
+                loader: 'css-loader',
```

Comment:
I'm assuming that code came from the css-modules webpack-demo page?
Also this might be worth checking out, for both of us https://github.com/gajus/react-css-modules